tools/functional-tester/etcd-tester/lease_stresser.go

```diff
@@ -326,9 +328,9 @@ func (ls *leaseStresser) keepLeaseAlive(leaseID int64) {
 			ls.aliveLeases.remove(leaseID)
 			return
 		}
-		// update lease's renew time
+		// renew lease timestamp only if lease is present
 		plog.Debugf("keepLeaseAlive renew lease %v", leaseID)
```

Comment:
I meant, if lease is not present, we don't keep-alive this leaseID, right? 

```go
if ok {
   plog.Debugf("keepLeaseAlive renew lease %v", leaseID)
   al.leases[leaseID] = t
}
```

?

Either way, lgtm.